client/post-editor/editor-ground-control/index.jsx

```diff
@@ -187,6 +187,18 @@ export class EditorGroundControl extends PureComponent {
 		return this.props.translate( 'Saved' );
 	}
 
+	getE2ESaveStatus() {
```

Comment:
The logic here looks exactly the same as in `getSaveStatusLabel`, except for translation of labels. I wonder if we could refactor it somehow to avoid repetition? The only thing that crossed my mind was to make `translate`, a parameter of `getSaveStatusLabel`, so we can override it with `identity` from `lodash` when we don't want translation to take place. For example:

```
<span
     className="editor-ground-control__save-status"
     data-e2e-status={ this.getSaveStatusLabel( identity ) }
 >
     { this.getSaveStatusLabel( this.props.translate ) }
 </span>
```

I'm not familiar with e2e code so I'm not sure if there is a more idiomatic way to do this.